src/Modal/Modal.test.js

```diff
@@ -149,9 +149,9 @@ describe('<Modal />', () => {
                     title='Title' />
             );
 
-            expect( component.find('div.fd-overlay--modal').getDOMNode().attributes[
+            expect( component.props()[
```

Comment:
This wasn't necessary in the original pr, but was failing all of a sudden. I'm not sure what happened.

Reply:
I'm going to investigate further - just keeping this pr up so we don't lose this work.